detections/application/splunk_risky_command_abuse_ml_detection.yml

```diff
@@ -0,0 +1,67 @@
+name: Splunk Risky Command Abuse ML Detection
```

Comment:
update detection name to: "Splunk Command and Scripting Interpreter Risky SPL MLTK"

Reply:
updated